index.bs

```diff
@@ -224,7 +224,7 @@ following steps:
  <li>Run [=retrieve the client hints set=] with |settingsObject|.
  <li>For each [=client hints token=] |lowEntropyHint| in the registry's [=low entropy hint table=], [=set/append=] |lowEntropyHint| to |hintSet|.
  <li>If |request|'s [=request/client=] is not null, then for each [=client hints token=] |requestHint| in
-|request|'s [=environment settings object/client hints set=], [=set/append=] |requestHint| to
+|settingsObject|'s [=environment settings object/client hints set=], [=set/append=] |requestHint| to
 |hintSet|.
```

Comment:
You probably want to conditionalize the "append value to the header list" below, normatively indicating that user agents are free to not do that. Having that be part of the privacy section is great, but may not be sufficient.